node-red-contrib-bot-message/bot-send-card.js

```diff
@@ -552,86 +656,88 @@ const buildReplyAdaptiveCard = (RED, node, locale, data, config, reply) => {
+            {Title}
+            \n\n
+            {Subtitle1} (this is Section1)
+            - {some text}
+            - {some other text} 
+            \n\n
+            {Subtitle2} (this is Section2)
+            - {some text}
+            - {some other text}
+            \n\n 
+            {Subtitle3} (this is Section3)
+            - {some text}
+            - {some other text}
         */
-        const attrIndex = tmp.lastIndexOf('\n');
-        tmp = textToShow.substring(0, attrIndex);
-
-        // primary card
-        buildAdaptiveCardJson(tmp, reply.body, separator, textButtonMarker, textButtonPrefix);
-        if (attach) addImageToAdaptiveCard(attach, attachSize, imageHorizontalAlignment, imageWidth, imageHeight, reply.body);
-
-        // expandable card
-        reply.actions = [];
-        tmp = textToShow.substring(attrIndex);
-        reply.actions.push({"type": "Action.ShowCard", "title": titleShowCardAction, "card": {"type": 'AdaptiveCard', "body": []}});
-        buildAdaptiveCardJson(tmp, reply.actions[0].card.body, separator, textButtonMarker, textButtonPrefix);
-    } else {
-        // otherwise show all the text within the primary card, no expandable card
-        buildAdaptiveCardJson(textToShow, reply.body, separator, textButtonMarker, textButtonPrefix);
-        if (attach) addImageToAdaptiveCard(attach, attachSize, imageHorizontalAlignment, imageWidth, imageHeight, reply.body);
+        if (!separator) {
+            // if not defined value, then consider sections are surrounded by '\n\n'.
+            separator = "\n\n";
+        }
+
+        if (!displayedTextSize) {
+            // if not defined value, then don't wrap, display up to 100k characters.
+            displayedTextSize = 100000;
+        }
+
+        textButtonMarker = helper.getContextValue(RED, node, data, textButtonMarker || '- ', 'str');
+        textButtonPrefix = helper.getContextValue(RED, node, data, textButtonPrefix || '', config.textButtonPrefixType || 'str');
+
+        titleShowCardAction = helper.getContextValue(RED, node, data, titleShowCardAction || 'More', config.titleShowCardActionType || 'str');
+        titleShowCardAction = marshall(locale, titleShowCardAction,  data, '');
+
+        title = helper.getContextValue(RED, node, data, title || '', config.titleTypeAdaptiveCard || 'str');
+        title = marshall(locale, title,  data, '');
+
+        attach = helper.getContextValue(RED, node, data, attach || '', config.attachTypeAdaptiveCard || 'str');
+        attach = marshall(locale, attach,  data, '');
+
+        attachSize = helper.getContextValue(RED, node, data, attachSize, 'str');
+        imageHorizontalAlignment = helper.getContextValue(RED, node, data, imageHorizontalAlignment, 'str');
+        imageWidth = helper.getContextValue(RED, node, data, imageWidth, 'str');
+        imageHeight = helper.getContextValue(RED, node, data, imageHeight, 'str');
+
+        reply.type = 'AdaptiveCard';
```

Comment:
regression on line 705 and 707

line 705 should be `reply.type = CARD_CONST.CARD_ADAPTIVECARD;`
line 707 should be removed